src/llmcompressor/transformers/finetune/runner.py

```diff
@@ -68,8 +68,8 @@ def populate_datasets(self, processor: Processor, add_labels: bool = True):
         :param processor: processor or tokenizer to use for dataset tokenization
         :param add_labels: if True, add labels column to dataset splits
         """
+        self.processor = processor  # TODO: pass processor into init instead of this fn
```

Comment:
if this processor is the same as the model processor, it will be accessible by self.model_args.processor.

Reply:
At this location, `processor is not self.model_args.processor`. We could change this flow in the future, but for now we separate `processor` and `self.model_args.processor` within text_generation.py in order to preserve `self.model_args.processor` being a string (if the user passed a string)